### PR TITLE
Added the Firestorm base package list

### PR DESCRIPTION
**Pull request by Mailaender**
for https://github.com/OpenRA/OpenRA/pull/11787.

### CODE REVIEW

**Comment by obrakmann** [comment]
:+1:

File: content/packages/fs-mirrors.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+http://openra.ppmsite.com/fs-base.zip
```
Comment (pchote) [inline content/packages/fs-mirrors.txt]: For the sake of a consistent naming scheme please call this `ts-fs-base.zip` or `ts-expand-base.zip`.  This keeps each mod's packages grouped logically on the servers.

**Comment by pchote** [comment]
If you want to split out only the fs-specific files in https://github.com/OpenRA/OpenRA/pull/11787 (which is a good idea) then we should hold on this until that's done.

**Comment by Mailaender** [comment]
Would make testing a bit easier (I just need to replace the ZIP to make changes) but we can also block this. Doesn't really matter that much.

File: content/packages/fs-mirrors.txt (hunk 1 above)
Comment (LipkeGu) [inline content/packages/fs-mirrors.txt]: ts-expand.zip seems and sounds logical :)

**Comment by LipkeGu** [comment]
```http://81.169.246.181/ts-expand.zip```

can you add this to the mirror-list? =)

**Comment by Mailaender** [comment]
I can't open it:

image

:disappointed:

**Comment by pchote** [comment]
Lets wait until we have the final content zip before we start spreading it around mirrors.

**Comment by LipkeGu** [comment]
the source is broken =( http://openra.ppmsite.com/fs-base.zip

**Pull request closed without merging**